src/nativeMain/kotlin/org/intellij/markdown/html/CommonDefsImplNative.kt

```diff
@@ -21,7 +21,6 @@ actual fun isWhitespace(char: Char): Boolean {
     return char == 0.toChar() || char.isWhitespace()
 }
 
-@SharedImmutable
```

Comment:
`SharedImmutable` isn't defined on the wasm target, but it's been a deprecated no-op since 1.9.0 anyway, so there's no harm in removing them.